document/core/exec/runtime.rst

```diff
@@ -666,6 +675,16 @@ In order to specify the reduction of :ref:`branches <syntax-instr-control>`, the
 
 This definition allows to index active labels surrounding a :ref:`branch <syntax-br>` or :ref:`return <syntax-return>` instruction.
 
+In order to be able to break jumping over exception handlers and caught exceptions, we must allow for these new structured administrative control instructions to appear after labels in block contexts, by extending block context as follows.
+
+.. math::
+   \begin{array}{llll}
+   \production{(block contexts)} & \XC^{k} &::=& \handler~\XB^k~\END \\
```

Comment:
Where is `\handler` defined?
```suggestion
   \production{(control contexts)} & \XC^{k} &::=& \handler~\XB^k~\END \\
```

Reply:
@rossberg 

> Where is `\handler` defined?

I meant to define `\handler` on line 552, in subsection `syntax-handler` in section `Stack` of this file. Currently it's still called `\X{handler}` there, even though I have added `\handler` to `util/macros.def`. Switching the forgotten `\X{handler}` to `\handler` in the next commit.  

@aheejin 

> Sphinx question: What does `production` do Sphinx?

This seems to be the way production rules are defined, perhaps @rossberg can explain this better or give more details?

> (And what are control contexts?)

I meant for control contexts to mean the parts of block contexts that are control flow (exception) handlers, but am switching to @rossberg 's suggestion above in the next commit. 
(I didn't add this suggestion automatically to not "resolve" this discussion without answering).

Reply:
@aheejin, there is a style of laying out grammars with a short title for each non-terminal, like so:
```
(types)        T ::= Int | Bool
(expressions)  E ::= E + E
                  |  if E then E else E
```
The `production` macro was intended to allow creating this, but is currently defined to expand to the empty string, in order to save horizontal space.

Reply:
@ioannad It looks @rossberg's suggestion in https://github.com/WebAssembly/exception-handling/pull/180#discussion_r820975483 hasn't been committed yet.

Reply:
Thanks again @aheejin, I am starting to worry a commit got entirely dropped somehow.